### PR TITLE
Print the bindings a worker has access to during `dev` and `publish`

### LINKED ISSUE #509: We should print out what bindings are being used in dev and publish, and what effect that will have.

### PR DESCRIPTION
**Pull request by caass**
It can be helpful for a user to know exactly what resources a worker will have access to and where they can access them, so we now log the bindings available to a worker during `wrangler dev` and `wrangler publish`.

Will rendering the `<Dev />` component interfere with the logged output during `wrangler dev`?

Closes #509

### CODE REVIEW

**Comment by caass** [comment]
I think it could be worth refactoring to take advantage of the existing [`CfWorkerInit[bindings]`](https://github.com/cloudflare/wrangler2/blob/print-bindings/packages/wrangler/src/worker.ts#L159) type

File: packages/wrangler/src/config/index.ts [hunk 1]
```diff
@@ -61,3 +62,164 @@ export function findWranglerToml(
...
+      })),
+    });
+  }
+
+  if (output.length === 0) {
+    return;
+  }
+
+  const message = [
+    `Your worker has access to the following:`,
```
Comment (petebacondarwin) [inline packages/wrangler/src/config/index.ts:213]: How about?

```suggestion
    `Your worker has access to the following bindings:`,
```

File: packages/wrangler/src/publish.ts [hunk 2]
```diff
@@ -493,6 +494,13 @@ export default async function publish(props: Props): Promise<void> {
       usage_model: config.usage_model,
     };
 
+    const withoutStaticAssets = {
+      ...bindings,
+      kv_namespaces: config.kv_namespaces,
+      text_blobs: config.text_blobs,
```
Comment (petebacondarwin) [inline packages/wrangler/src/publish.ts:500]: Clever

**Review by petebacondarwin — APPROVED** [review]
Love it!

**Pull request merged**